src/Widgets/Tooltip.vala

```diff
@@ -55,16 +48,33 @@ public class Gala.Tooltip : Clutter.Actor {
     public float max_width;
 
     static construct {
-        var scale = InternalUtils.get_ui_scaling_factor ();
+        var dummy_label = new Gtk.Label ("") {
```

Comment:
Even though this is really convenient, it'll crash on Wayland, like it happens now with the tile outline preview.

Reply:
I guess it would crash anywhere we're using the Gtk foreign drawing API then? Because we're using it in several places to get accent colors

Reply:
Yes, using Gtk in the compositor causes the problem, not the foreign drawing API in particular. We have the same issue with the shadows, for example. I'd personally avoid it.